still active?

I'm cleaning up the https://docs.influxdata.com/influxdb/v1.0/tools/api_client_libraries/ page and removing those libraries that aren't compatible with InfluxDB 1.0. 

I'll remove this project from that page on October 15th unless you have plans to update for 1.0 compatibility. Thanks!

I would also be interested in the state of things. The current version doesn't even seem to compile properly against OpenSSL (`undefined reference to SSLv3_method`).

If the library is unmaintained, a note in the README would help. I only noticed these problems after finishing the implementation of a stats logigng tool, unfortunately :(

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -10,6 +10,8 @@ readme = "README.md"
 keywords = ["influx", "influxdb"]
 license = "MIT"
 
+maintenance = { status = "looking-for-maintainer" }
+
 [dependencies]
 hyper = { version = "0.12" }
 tokio = "0.1"

